### PR TITLE
exec() and execstart() in ContainerManager

### PR DESCRIPTION
**Pull request by joelwurtz**
Replace #90

### CODE REVIEW

File: docs/container.md [hunk 1]
```diff
@@ -203,4 +203,21 @@ $ports = new Docker\PortCollection(80, 22);
 $ports->add(42);
 
 $container->setExposedPorts($ports);
-```
\ No newline at end of file
+```
+
+## Exec: running command within an existing running container.
```
Comment (hacfi) [inline docs/container.md]: I think `running a command` or `run command` sounds better.

**Comment by hacfi** [comment]
:+1: Really look forward to this feature @Boran @joelwurtz

File: docs/container.md (hunk 1 above)
Comment (joelwurtz) [inline docs/container.md]: Fixed, thanks

**Comment by joelwurtz** [comment]
Merged, Thanks @Boran for the work on exec !

**Comment by Boran** [comment]
Thanks for the review and commit.
I had a look at your changes, thanks for adding the test. Adding the callback is useful too.
In the doc changes you made "$logger = new Psr\Log();" wont work out of the box, composer.json would need to be adapted to load it? 
Although it might be a nice idea to add that as a dependency, logging examples for debugging could then be added.

**Pull request merged**